doc/book/admin/disaster_recovery.rst

```diff
@@ -1,126 +1,162 @@
-:ref:`replication cluster<replication-topologies>`
+The minimal fault-tolerant Tarantool configuration would be a :ref:`replica set <replication-architecture>`
 that includes a master and a replica, or two masters.
+The basic recommendation is to configure all Tarantool instances in a replica set to create :ref:`snapshot files <index-box_persistence>` on a regular basis.
 
-The basic recommendation is to configure all Tarantool instances in a cluster to
-create :ref:`snapshot files <index-box_persistence>` at a regular basis.
+Here are action plans for typical crash scenarios.
 
-Here follow action plans for typical crash scenarios.
 
 .. _admin-disaster_recovery-master_replica:
 
---------------------------------------------------------------------------------
 Master-replica
---------------------------------------------------------------------------------
+--------------
 
-Configuration: One master and one replica.
+.. _admin-disaster_recovery-master_replica_manual_failover:
 
-Problem: The master has crashed.
+Master crash: manual failover
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Your actions:
+**Configuration:** master-replica (:ref:`manual failover <replication-master_replica_bootstrap>`).
 
-1. Ensure the master is stopped for good. For example, log in to the master
-   machine and use ``systemctl stop tarantool@<instance_name>``.
+**Problem:** The master has crashed.
 
-2. Switch the replica to master mode by setting
-   :ref:`box.cfg.read_only <cfg_basic-read_only>` parameter to *false* and let
-   the load be handled by the replica (effective master).
+**Actions:**
 
-3. Set up a replacement for the crashed master on a spare host, with
-   :ref:`replication <cfg_replication-replication>` parameter set to replica
-   (effective master), so it begins to catch up with the new master’s state.
-   The new instance should have :ref:`box.cfg.read_only <cfg_basic-read_only>`
-   parameter set to *true*.
+1.  Ensure the master is stopped.
+    For example, log in to the master machine and use ``tt stop``.
 
-You lose the few transactions in the master
-:ref:`write ahead log file <index-box_persistence>`, which it may have not
-transferred to the replica before crash. If you were able to salvage the master
-.xlog file, you may be able to recover these. In order to do it:
+2.  Configure a new replica set leader using the :ref:`<replicaset_name>.leader <configuration_reference_replicasets_name_leader>` option.
 
-1. Find out the position of the crashed master, as reflected on the new master.
+3.  Reload configuration on all instances using :ref:`config:reload() <config-module>`.
 
-   a. Find out instance UUID from the crashed master :ref:`xlog <internals-wal>`:
+4.  Make sure that a new replica set leader is a master using :ref:`box.info.ro <box_introspection-box_info>`.
 
-      .. code-block:: console
+5.  Remove a crashed master from a replica set as described in :ref:`Removing instances <replication-remove_instances>`.
```

Comment:
Shouldn't we link `how-to/replication/repl_bootstrap/#removing-an-instance-from-the-cluster-space` instead?

I mean, if the replacement instance from the next (sixth) step is supposed to have the same name as the old one, we need to just drop an entry from `_cluster` and start the instance somewhere again. (It also means that `Adding instances` section is not very relevant.)

So, the minimal working steps instead of these 5-6 ones are the following:

5. Remove the instance from the _cluster space.
6. Start the instance again on a spare host.